cidc_api/config/settings.py

```diff
@@ -55,15 +55,16 @@
 ALGORITHMS = ["RS256"]
 
 ### Configure GCP ###
-GOOGLE_CLOUD_PROJECT = environ.get("GOOGLE_CLOUD_PROJECT")
-GOOGLE_UPLOAD_BUCKET = environ.get("GOOGLE_UPLOAD_BUCKET")
-GOOGLE_UPLOAD_TOPIC = environ.get("GOOGLE_UPLOAD_TOPIC")
-GOOGLE_DATA_BUCKET = environ.get("GOOGLE_DATA_BUCKET")
-GOOGLE_UPLOAD_ROLE = environ.get("GOOGLE_UPLOAD_ROLE")
+GOOGLE_CLOUD_PROJECT = environ["GOOGLE_CLOUD_PROJECT"]
```

Comment:
We want to get `KeyError`s if these environment variables aren't set, because they're critical to the API functioning correctly.